### PR TITLE
feat: support patch crds conversion

### PR DESCRIPTION
**Pull request by JarHMJ**
Signed-off-by: huangminjie <minjie.huang@daocloud.io>

support this pr https://github.com/kubevela/kubevela/pull/3890

### CODE REVIEW

**Comment by msftgits** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/oam-dev/kube-webhook-certgen?pullRequest=5) <br/>All CLA requirements met.

File: cmd/patch.go [hunk 1]
```diff
@@ -38,6 +40,18 @@ func prePatchCommand(cmd *cobra.Command, args []string) {
...
 func patchCommand(_ *cobra.Command, _ []string) {
+	var crdsConversion []*k8s.CRDConversion
+	for i := 0; i < len(cfg.crds); i++ {
+		var Conversion k8s.CRDConversion
+		err := json.Unmarshal([]byte(cfg.crds[i]), &Conversion)
+		if err != nil {
+			log.Fatalf("CRD Conversion: %s Unmarshal err: %s", cfg.crds[i], err)
+		}
+		crdsConversion = append(crdsConversion, &Conversion)
+		fmt.Printf("%+v\n", crdsConversion[i])
```
Comment (wonderflow) [inline cmd/patch.go]: let's use log.Println

File: cmd/root.go [hunk 2]
```diff
@@ -3,10 +3,11 @@ package cmd
 import (
 	"os"
 
-	admissionv1 "k8s.io/api/admissionregistration/v1"
+	"github.com/jet/kube-webhook-certgen/pkg/k8s"
```
Comment (wonderflow) [inline cmd/root.go:6]: this is the same repo which we forked from, we should merge this code if not exist inside.

Comment (JarHMJ) [reply cmd/root.go:6]: yes，I add the code by myself

File: cmd/patch.go (hunk 1 above)
Comment (JarHMJ) [reply cmd/patch.go]: I have removed this line

**Review by wonderflow — APPROVED** [review]
great job

**Pull request merged**